dev/CMakeLists.txt

```diff
@@ -109,3 +109,15 @@ target_link_libraries(${EXE_NAME}
   ${SDL_LIBRARIES}
   ${ADDITIONAL_LIBRARIES}
   ${OPENGL_LIBRARIES})
+
+if(UNIX)
+   include(GNUInstallDirs)
+   set(BINDIR ${CMAKE_INSTALL_FULL_BINDIR})
+   set(DATADIR ${CMAKE_INSTALL_FULL_DATADIR})
```

Comment:
`DATADIR` is too to clash as a preprocessor symbol, maybe `PLATFORM_DATADIR` ?

Reply:
Right, done.